Qt1 385 rpl bulk upload validation messages durationreducedby

alireid opened the pull request.

Pauljgraham reviewed (changes requested):
Looks great. One tiny issue on the tests. We cover all the failures, but not the successful routes. Can we add a test to cover the success values. The other issue is we seem to have two PRs on this ticket for the same repo

alireid commented:
> Looks great. One tiny issue on the tests. We cover all the failures, but not the successful routes. Can we add a test to cover the success values. The other issue is we seem to have two PRs on this ticket for the same repo

Thanks Paul. Deleted the other PR. Added test with values that will produce no errors (aka happy path) and utilise ValidateNoErrorsFound
